alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/AlertPagedDetails.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * alert-common
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.common.rest.model;
+
+import java.util.List;
+
+public class AlertPagedDetails<M> {
+
+    // FIXME we should use terminology based on "offset" and "limit" which are standard REST API paging terms
+    private final int totalPages;
+    private final int currentPage;
+    private final int pageSize;
+    private final List<M> models;
+
+    public static <M> AlertPagedDetails<M> EMPTY_PAGE() {
```

Comment:
I don't think static methods need to be all caps. Think `List.of`, `Collections.singletonList(...)`, etc.